### ISSUE TITLE
[journaux régionaux] Le Bien Public et L'Yonne Républicaine

### ISSUE THREAD

**Issue by villisek** [lovasoa/ophirofox#205]
Bonjour, 

Serait-il possible d'ajouter le support de ces deux journaux, présents dans Europresse ?
- Le Bien Public
Journal de Côte-d'Or
https://www.bienpublic.com/

- L'Yonne Républicaine
Journal de l'Yonne
https://www.lyonne.fr/ 
Leur site est similaire à celui de la Montagne
Europresse distingue la version kiosque de la version web du journal, donc certains articles peuvent peut-être apparaître en double dans la recherche.

Merci :)

**Comment by lovasoa** [comment]
Oui, cela doit être possible ! Les contributions sont les bienvenues !

**Comment by lovasoa** [comment]
Dans ce cas je pense que cela ne fait pas tellement de sens de l'inclure...

**Comment by Write** [comment]
Je clos pour le moment, vu le manque d'articles disponibles sur Europresse.
Ne pas hésiter à rouvrir un ticket si la situation change.
